features/checkout/preventing_claiming_cart_of_logged_in_user.feature

```diff
@@ -0,0 +1,25 @@
+@checkout
+Feature: Checking out as guest with existing email, while email owner starts a new cart session
+    In order to make the checkout cart available only for user who owns the cart
+    As a Logged in user
+    I want to be able to checkout with my previous cart when someone used my email in checkout
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt" priced at "$20.00"
+        And the store has a product "Kotlin T-Shirt" priced at "$30.00"
+        And the store has a product "Symfony T-Shirt" priced at "$100.00"
+        And the store has a product "Sylius T-Shirt" priced at "$150.00"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a user "robb@stark.com" identified by "KingInTheNorth"
+        And I am logged in as "robb@stark.com"
+
+    @ui @javascript
+    Scenario: Preventing anonymous user claiming logged in user's cart
+        Given I have product "PHP T-Shirt" in the cart
+        When an anonymous user in another browser adds products "PHP T-Shirt" and "Kotlin T-Shirt" to the cart
+        And he completes addressing step with email "robb@stark.com" and "United States" based billing address
+        And I view cart on my browser
```

Comment:
`And I view cart my cart`? `session` is quite technical

Reply:
thats why i changed it to browser as session is too technical

Reply:
_I view cart **in** my browser_

Reply:
On the other hand... `in my browser` is obvious, how can it be the other way? XD `And I view my cart` is taken? Can we think about something not technical and not obvious? 😄 `And I come back to my previous cart`?